src/utils/constants.js

```diff
@@ -1 +1,8 @@
+import { useContext, createContext } from 'react';
+export const AuthContext = createContext();
+
 export const URL_TODOS = 'http://localhost:3030/api/todos/';
+
+export const useAuth = () => {

```

Comment:
i would suggest creating a folder called "hooks" and a file called "useAuth" in it. and export the useAuth hook from there.
it's not so obvious on this project yet, but in the future if there would be a lot of hooks, it's good to have them be all in one place to find them easy